codex-ui/src/vue/components/context-menu/ContextMenu.vue

```diff
@@ -58,6 +61,7 @@ withDefaults(
 
   &__input {
     padding: var(--v-padding) var(--h-padding);
+    color: var(--base--text-secondary)
```

Comment:
it seems redundant, placeholder is expected instead

Reply:
you mean move Icon in placeholder?